packages/app/tsconfig.check.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "exclude": ["codesandbox-browserfs/*"],
   "compilerOptions": {
-    "allowJs": false
+    "allowJs": false,
+    "skipLibCheck": true
```

Comment:
@SaraVieira i've fixed the `date-fns` typings error enable this setting, but i don't know if is ok for the codesandbox setup

Reply:
What was the typing errors? Should not be any issue using `date-fns` with Typescript?

Reply:
yes, it's one error with the `date-fns` typing definition